src/components/AmountFilter.jsx

```diff
@@ -11,20 +11,34 @@ export default function AmountFilter(props) {
   return (
     <Grid container>
       <Grid item sx={{ m: 1 }}>
```

Comment:
The slider isn't center aligned with the textfields. You can [auto lay it out](https://mui.com/components/grid/#auto-layout) so that there is `space-between` the field, element and dash and get rid of the margins here. Instead of margins use Grid spacing and set the `xs` property accordingly.

If you do need to add margin/padding add it to the Grid container or the `AccordionDetails` element so that the entire filter area (including the slider below) gets spacing to the sides.